wishlist/wishlist.css

```diff
@@ -27,7 +27,30 @@
 .wishlist-ratings {
     margin-top: 5rem;
 }
-
+.logout-btn {
+    color: var(--orange);
+    border: solid 2px var(--orange);
+    background-color: var(--white);
+    font-weight: normal;
+    font-size: 1rem;
+    margin-top: 0.8rem;
+    margin-left: 1rem;
```

Comment:
You can make utility class of "font-size: 1rem", "margin-left: 1 rem", etc. The reason for saying this is that these properties are the most common ones. You can make separate classes for them and use them whenever you want in any project. If you have already made them, then you could use them here.

Reply:
Ok, will take into consideration from now on, thanks for suggesting